source/includes/api/_dataset.md

```diff
@@ -74,81 +76,140 @@ curl -X GET https://api.resourcewatch.org/v1/dataset
-published     |                                                                              | `true`or `false`
-env           | If the dataset is in the staging, preproduction environment or in production one          | `staging`, `production` or `preproduction`. Defaults to `production`
-overwritted   | If the data can be overwritten (only for being able to make dataset updates) | `true`or `false`
-verify        | If this dataset contains data that is verified using blockchain              | `true`or `false`
-protected     | If it's a protected layer                                                    | `true`or `false`
-geoInfo       | If it contains interceptable geographical info                               | `true`or `false`
-subscribable  | If the dataset is subscribable (i.e. contains a non-empty object in the subscribable field) | `true` or `false`
+### Filters
 
 > Filtering datasets
 
 ```shell
 curl -X GET https://api.resourcewatch.org/v1/dataset?name=birds&provider=cartodb
 ```
 
-> For inclusive filtering with array props use '@'
+> Matching vocabulary tags
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/dataset?app=gfw@rw@prep
+curl -X GET https://api.resourcewatch.org/v1/dataset?vocabulary[legacy]=umd
 ```
 
-### Sorting
+The dataset list provides a wide range of parameters that you can use to tailor your dataset listing. Most of these parameters reflect fields you'll find in a dataset itself, while others are convenience filters for things like user role or favourites. These parameters can be combined into a complex `and` logic query.
 
-#### Basics of sorting
+Here's the comprehensive list of filters supported by datasets:
+ 
 
-The API currently supports sorting by means of the `sort` parameter. Sorting can be done using any field from the dataset, as well as `user.name` and `user.role` (sorting by user data is restricted to ADMIN users).
+Filter         | Description                                                                  | Type        | Expected values
+-------------- | ---------------------------------------------------------------------------- | ----------- | -------------------------------------------------------------------------------
+name           | Filter returned datasets by the name of the dataset.                         | String      | any valid text
+slug           | Filter returned datasets by the slug of the dataset.                         | String      | any valid text
+type           | Filter returned datasets by dataset type.                                    | String      | any valid text
+subtitle       | Filter returned datasets by dataset subtitle.                                | String      | any valid text
+application    | Applications associated to this dataset.                                     | Array       | any valid text
+applicationConfig |                                                                           | Object      | `true` or `false`
```

Comment:
Why is this `Object` and not `Boolean`?

Reply:
Should be boolean. The field itself is an object, but the way the filters are built is that Objects are filtered by a bool that says if they are empty or not.